Source/Bugsnag.h

```diff
@@ -200,7 +200,8 @@ static NSString *_Nonnull const BugsnagSeverityInfo = @"info";
  *
  * @param capacity max number of breadcrumb log messages to send
  */
-+ (void)setBreadcrumbCapacity:(NSUInteger)capacity;
++ (void)setMaxBreadcrumbs:(NSUInteger)capacity;
```

Comment:
This is not on the Bugsnag interface any more according to the spec, and should be on Configuration instead.